scripts/setup-test-data/src/setup_test_data/cli.py

```
@@ -0,0 +1,422 @@
+                assets=item.get("assets", []),
+            )
+        )
+    return releases
+
+
+@retry(
+    wait=wait_exponential(multiplier=1, min=2, max=10),
+    stop=stop_after_attempt(3),
+)
+def download_asset(
+    client: httpx.Client, url: str, dest: Path, quiet: bool = False
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)